mylist.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*initial)
+    @initial = initial
+  end
+
+  def each(&block)
```

Comment:
- [ ] As the requirement mentioned you should implement the `#each` method that yields all members of `@list` in succession without the use of the ruby `.each` method